fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudClusterChecker.java

```diff
@@ -449,6 +447,8 @@ private void getCloudBackends() {
             // local - remote > 0, drop bes from local
             checkToDelCluster(remoteClusterIdToPB, localClusterIds, clusterIdToBackend);
 
+            clusterIdToBackend = cloudSystemInfoService.getCloudClusterIdToBackend();
```

Comment:
Is this a bug fix not related to smooth upgrade?

Reply:
yes,  not related to smooth upgrade